### PR TITLE
[PMS-540] Set user_id and accounts_user_id in session

### PR DESCRIPTION
**Pull request by pankopiotr**
https://bookingsync.atlassian.net/browse/PMS-540

### CODE REVIEW

File: .github/workflows/ruby.yml [hunk 1]
```diff
@@ -19,7 +19,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        ruby-version: ['2.6', '2.7']
+        ruby-version: ['3.1', '3.2', '3.3']
```
Comment (pyromaniac) [inline .github/workflows/ruby.yml]: Add 2.7 and 3.0, they are not EOL yet and used in our apps

Comment (pankopiotr) [reply .github/workflows/ruby.yml]: 2.7 is eol, 3.0 is eol in 2 months, nobody will touch this app in months.

If we're still using them, we have bigger problem than this

**Review by Azdaroth — APPROVED** [review]
💪 

Before merging though, let's make sure that the Core gets deployed first. Alternatively, we could use something safer than `fetch` so that it doesn't blow up when the attribute is not present.

**Comment by pankopiotr** [comment]
@Azdaroth Let's wait for Core, I would rather have explicit errors that something went wrong.

**Review by pyromaniac — APPROVED** [review]
LG

**Comment by smily-ivan** [comment]
btw, bump version? Or you do it in separate PR?

**Comment by pyromaniac** [comment]
@smily-ivan @Azdaroth is going to do it soon

**Comment by Azdaroth** [comment]
@pyromaniac  @smily-ivan  released

**Pull request merged**